pkgs/shells/rc-9front/default.nix

```diff
@@ -38,6 +48,19 @@ stdenv.mkDerivation {
   passthru = {
     shellPath = "/bin/rc";
     updateScript = unstableGitUpdater { shallowClone = false; };
+    tests = {
+      simple = runCommand "rc-test" { } ''
+        ${rc-9front}/bin/rc -c 'nl=`{echo} && \
+          res=`$nl{for(i in `{seq 1 10}) echo $i} && \
+          echo -n $res' >$out
```

Comment:
Please use nativeBuildInputs instead of hardcoding the path to the binaries

Reply:
Correct me if I'm wrong but I don't believe `nativeBuildInputs` are added to the package tests, when I tried adding `nawk` to nativeBuildinputs and removing the direct paths I was getting failures.